File: java-serializers/java-serializer-plugin-test-json-java/src/main/java/online/codeisfun/plugins/serializers/test/Data1.java
package online.codeisfun.plugins.serializers.test;



import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import online.codeisfun.plugins.serializers.CIFSerializable;

import java.util.Map;
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@CIFSerializable
public class Data1 {
    private Integer a;
    private String b;
    private Map<String, String> c;
    private Map<String, OtherClass1> d;
}

File: java-serializers/java-serializer-plugin-test-kryo-kotlin/src/main/kotlin/online/codeisfun/plugins/serializers/test/Data1.kt
package online.codeisfun.plugins.serializers.test

import online.codeisfun.plugins.serializers.CIFSerializable

@CIFSerializable
data class Data1(
    val a: Int? = null,
    val b: String? = null,
    val c: Map<String, String>? = null,
    val d: Map<String, OtherClass1>? = null,
)
